src/couch_replicator_scheduler.erl

```diff
@@ -473,6 +500,56 @@ update_history(Job, Type, When, State) ->
     Job#job{history = History1}.
 
 
+-spec update_running_jobs_stats() -> ok.
+update_running_jobs_stats() ->
+    Acc0 = #stats_acc{now = os:timestamp()},
+    AccR = ets:foldl(fun stats_fold/2, Acc0, ?MODULE),
+    #stats_acc{
+        pending_t = PendingSumUSec, pending_n = PendingN,
+        running_t = RunningSumUSec, running_n = RunningN,
+        crashed_t = CrashedSumUSec, crashed_n = CrashedN
+    } = AccR,
+    PendingAvg = avg(PendingSumUSec, PendingN),
+    RunningAvg = avg(RunningSumUSec, RunningN),
+    CrashedAvg = avg(CrashedSumUSec, CrashedN),
+    couch_stats:update_gauge([couch_replicator, jobs, pending], PendingN),
+    couch_stats:update_gauge([couch_replicator, jobs, running], RunningN),
+    couch_stats:update_gauge([couch_replicator, jobs, crashed], CrashedN),
+    couch_stats:update_gauge([couch_replicator, jobs, avg_pending], PendingAvg),
+    couch_stats:update_gauge([couch_replicator, jobs, avg_running], RunningAvg),
+    couch_stats:update_gauge([couch_replicator, jobs, avg_crashed], CrashedAvg),
+    ok.
+
+
+-spec stats_fold(#job{}, #stats_acc{}) -> #stats_acc{}.
+stats_fold(#job{pid = undefined, history = []}, Acc) ->
+    Acc;
```

Comment:
Agree but we don't have any history. Not sure what to do about it. So just skipped it.